remove_single_dimensional.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+
+
+def remove_single_dimensional(arr):
```

Comment:
What is arr? is it a numpy array, is it a shape? Please be more expressive with your variable names.
From a readers point of view it is not clear what this function is doing.